src/Game.elm

```diff
@@ -0,0 +1,22 @@
+module Game exposing (new, takeTurn)
+
+import Board exposing (Board, Mark(..), mark)
+
+type alias Game = {board : Board, currentPlayer : Mark}
+
+new : Board -> Game
+new board = 
+  {board = board, currentPlayer = X}
```

Comment:
Is player `X` always starting for any `board`?
What if the board received is not completely empty? What if it has a move already?